src/main/java/net/neoforged/neoforge/common/data/internal/NeoForgeLanguageProvider.java

```diff
@@ -445,6 +464,11 @@ private <T> void addColored(TagKey<T> baseTagKey, String pattern) {
         }
     }
 
+    private void add(FluidUnit unit, String singular, String plural) {
```

Comment:
You could save a little effort here with an overload that just appends `s` for plural.

Reply:
"mB" for millibucket isn't pluralized though, which means [unless i want to switch to writing it out as "millibucket"] i need two different methods and this method can't simply be called 'add' but needs to be something like 'addWithPluralS'. I don't think it's worth it.

Reply:
I said overload, you can keep around this signature too for the odd cases. Its a private method so the name is not too important, the two parameter add is overriding the plural